modules/@apostrophecms/doc-type/ui/apos/components/AposDocEditor.vue

```diff
@@ -184,30 +191,34 @@ export default {
     }
   },
   methods: {
-    async submit() {
-      apos.bus.$emit('busy', true);
+    submit() {
+      this.triggerValidation = true;
+      this.$nextTick(async () => {
+        if (!this.doc.hasErrors) {
+          apos.bus.$emit('busy', true);
+          let route;
+          let requestMethod;
+          if (this.docId) {
+            route = `${this.moduleOptions.action}/${this.docId}`;
+            requestMethod = apos.http.put;
+          } else {
+            route = this.moduleOptions.action;
+            requestMethod = apos.http.post;
+          }
 
-      let route;
-      let requestMethod;
-      if (this.docId) {
-        route = `${this.moduleOptions.action}/${this.docId}`;
-        requestMethod = apos.http.put;
-      } else {
-        route = this.moduleOptions.action;
-        requestMethod = apos.http.post;
-      }
+          try {
+            await requestMethod(route, {
+              busy: true,
```

Comment:
Pretty sure this busy: true is enough and we don't need the manual emitting of busy events here, but this is not new in this PR so I won't be a pain about it (:

Reply:
I can remove `apos.bus.$emit('busy', true);` if you think it it is useless.

Reply:
You were right. Removed